app/assets/javascripts/discourse/app/lib/autocomplete.js

```diff
@@ -233,11 +234,27 @@ export default function (options) {
         if (term) {
           let text = me.val();
 
+          // resync in case of drift (upload for example)
```

Comment:
I think this could be slightly more specific, like `(upload placeholder text in composer for example)`, otherwise I am not sure as a reader how uploads are related

Reply:
sure done